### PR TITLE
refactor: better error message for Facebook unauthorized

### PR DESCRIPTION
**Pull request by scmmishra**
This PR adds better error message when our machine overlords at Facebook returns unauthorized error

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

CleanShot 2023-09-18 at 18 59 34@2x

Tested locally, production testing pending

### CODE REVIEW

**Comment by scmmishra** [comment]
had to force push, the branch was dirty 🥲

File: app/javascript/dashboard/i18n/locale/en/inboxMgmt.json [hunk 1]
```diff
@@ -371,6 +371,8 @@
     "DETAILS": {
       "LOADING_FB": "Authenticating you with Facebook...",
       "ERROR_FB_AUTH": "Something went wrong, Please refresh page...",
+      "ERROR_FB_UNAUTHORIZED": "You're not authorized to perform this action. ",
+      "ERROR_FB_UNAUTHORIZED_HELP": "Please ensure you have access to the Facebook page with full control. You can read more about Facebook roles <a href=\" https://www.facebook.com/help/187316341316631\">here</a>.",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:375]: Can you please re-write this to Facebook specific language? Full Control seems like a generic term and the user wouldn't probably receive anything when they search for it on Google.

Comment (scmmishra) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:375]: @pranavrajs "Full Control" is the term they use. I tried finding a specific term, but couldn't

ref: https://www.facebook.com/business/help/582754542592549?id=418112142508425

Comment (scmmishra) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:375]: @pranavrajs Should I just use the word "Admin" here?

Comment (pranavrajs) [reply app/javascript/dashboard/i18n/locale/en/inboxMgmt.json:375]: Let us see how this goes.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**